python/docs/source/user_guide/pandas_on_spark/options.rst

```diff
@@ -227,24 +227,24 @@ This is conceptually equivalent to the PySpark example as below:
 Available options
 -----------------
 
-=============================== ============== =====================================================
+=============================== ============== ===================================================================
 Option                          Default        Description
```

Comment:
@itholic, actually this chart is generated from https://github.com/apache/spark/blob/master/python/pyspark/pandas/config.py#L114-L118. Can you use this codes to generate the chart?